po/core/kvirc/kvirc.pot

```diff
@@ -1834,19 +1834,19 @@ msgstr ""
 
 #: src/modules/ident/libkviident.cpp:403
 #: src/modules/ident/libkviident.cpp:423
-msgid "Identd accepting connection"
+msgid "Ident accepting connection"
```

Comment:
starting here, it should really be identd or ident service as above, but that might be too long?